utils/aws/aws.go

```diff
@@ -128,13 +130,18 @@ func FakeNewClientFunc(elbClient ELBv2APIClient) func() (Client, error) {
 }
 
 func (c *ClientAdapter) FindLoadBalancerByDNSName(ctx context.Context, dnsName string) (*elbv2types.LoadBalancer, error) {
-	lbOutput, err := c.ELBV2.DescribeLoadBalancers(ctx, &elbv2.DescribeLoadBalancersInput{})
-	if err != nil {
-		return nil, err
-	}
-	for _, lb := range lbOutput.LoadBalancers {
-		if lb.DNSName != nil && *lb.DNSName == dnsName {
-			return &lb, nil
+	paginator := elbv2.NewDescribeLoadBalancersPaginator(c.ELBV2, &elbv2.DescribeLoadBalancersInput{
+		PageSize: aws.Int32(300),
```

Comment:
can you add 300 in github.com/argoproj/argo-rollouts/utils/defaults and use it here?